src/series/Castlevania.yml

```diff
@@ -86,25 +82,35 @@ randomizers:
     updated-date: 1900-01-01
     added-date: 1900-01-01
 -   games:
-    - 'Castlevania: Symphony of the Night (PlayStation / Saturn)'
-    identifier: 3snowp7im's Area Randomizer
-    url: https://github.com/3snowp7im/SotN-Randomizer
-    comment: 'Includes a Richter converter for Wild Mouse''s item randomizer, and a standalone Maria item randomizer. Browser version here: <https://sotn.io/>'
-    updated-date: 2024-08-21
+    - 'Castlevania: Symphony of the Night'
+    identifier: "Castlevania: SOTN Randomizer"
+    url: https://symphonyrando.fun
```

Comment:
this url doesn't work https://symphonyrando.fun

Reply:
Needs the www after //. My bad, i forget that all the time because it was a recent requirement change

Reply:
seems like the actual URL is https://www.symphonyrando.fun/ with the www

Reply:
Yeah